lumispy/signals/common_luminescence.py

```diff
@@ -96,9 +96,10 @@ def scale_by_exposure(self, exposure=float("nan"), inplace=False):
         Parameters
         ----------
         exposure : float
-            Exposure time in s. If not given, the function tries to find
-            'exposure' or 'dwell_time' in the metadata (for the moment only at
-            Gatan specific nodes).
+            Exposure time in s. If not given, the function tries to use the
```

Comment:
Rename the `exposure` argument to `integration_time` so that it is consistent with the metadata specification?

Reply:
Yes, it would make sense, but I would not want to rename the function not to break the API and would probably allow `exposure` as backwards compatible Alias.

Reply:
Added deprecation warning.